### DISCUSSION TITLE
Setting HideClockOnSecondaryMonitors

### DISCUSSION THREAD

**Discussion by micheldiemer** [martinet101/ElevenClock#1283] [Q&A]
In the folder %userprofile%\.elevenclock I see that there is a file name HideClockOnSecondaryMonitors

I wand ElevenClock only on the main screen so please how do I enable "HideClockOnSecondaryMonitors" ? I have not seen this setting on the Graphical User Interface.

The setting "Do not display on this screen" is not work working very well. I work with 1, 2 or 3 screens and when I click on "Do not display on this screen" ElevenClock tends to dissappear even on the main screen if I don't plug a second screen.

**Comment by marticliment** [comment]
Hello,

The setting file you have found is deprecated and no longer used.

To hide clocks from certain monitors you need to blacklist them from the context menu:
![image](https://github.com/marticliment/ElevenClock/assets/53119851/51bf4829-3f90-43af-986b-16737d522385)

**Comment by micheldiemer** [comment]
Thank you for your reply.
It is not working very well. I have just done it right now. I have three monitors, blacklisted two of them. ElevenClock is displayed on the main monitor, that's fine.
Restart the computer. ElevenClock is on the second monitor, not on the main monitor.
This is why I was looking for another setting.
It is a laptop so I rebbot it frequently and also I plug different monitors on it depending on the place where I go.
Should I open another issue ?

Cordialement,
Michel DIEMER
Formateur en ***@***.*** 

    Le samedi 18 novembre 2023 à 10:18:45 UTC+1, Martí Climent ***@***.***> a écrit :  
 
 


Hello,

The setting file you have found is deprecated and no longer used.

To hide clocks from certain monitors you need to blacklist them from the context menu:

**Reply by marticliment** [reply]
Hello,

Yes, please open a new issue

**Comment by micheldiemer** [comment]
Thank you very much for replying.
Issue #1290 opened
Blacklist this monitor : wrong monitor blacklisted after reboot · Issue #1290 · marticliment/ElevenClock


| 
| 
| 
|  |  |

 |

 |
| 
|  | 
Blacklist this monitor : wrong monitor blacklisted after reboot · Issue ...

I have three monitors, blacklisted two of them. ElevenClock is displayed on the main monitor, that's fine. Reboo...
 |

 |

 |




Cordialement,
Michel DIEMER
Formateur en ***@***.*** 

    Le lundi 4 décembre 2023 à 15:46:33 UTC+1, Martí Climent ***@***.***> a écrit :  
 
 


Hello,

Yes, please open a new issue
